bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/widgets/Shell.java

```diff
@@ -479,20 +479,42 @@ void addToolTip (ToolTip toolTip) {
 	toolTips = newToolTips;
 }
 
-void adjustTrim () {
+/**
+ * Resize the shell based on the trim width and height. Trim width and height
+ * adjustments are read from 'user/.swt.trims.prefs' for each trim style. If
+ * such a file does not exist then it uses default values initialized.
```

Comment:
`then it uses default values initialized.` -> `then the trim values are queried from GTK`.